network/load-balancer/how-to/create-manage-acls.mdx

```diff
@@ -69,6 +69,6 @@ An **A**ccess **C**ontrol **L**ist (ACL) is a list of permissions that control w
 
```

Comment:
Regarding edit and delete ACL, both procedures have the step 2 saying: Click the Load Balancer you want to create ACLs for.
We shouldn't use create and instead adapt the verb to the actual procedure --> edit or delete

Reply:
done